Woof/Woof/View/OwnerFlow/OwnerOnboardingView.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+
+/// A view that onboards the owner and also collects mandatory information about them.
+struct OwnerOnboardingView: View {
+    /// The name of the owner.
+    @Binding var name: String
+
+    /// The surname of the owner.
+    @Binding var surname: String
+
+    /// The phone of the owner.
+    @Binding var phone: String
+
+    /// The owner's city.
+    @Binding var city: String
+
+    /// The address of the owner street and building
+    @Binding var address: String
+
+    /// A boolean indicating whether the save button should be enabled.
+    var isSaveButtonEnabled: Bool {
+        !name.isEmpty && !phone.isEmpty && !city.isEmpty && !address.isEmpty
+    }
+
+    var body: some View {
+        VStack {
+            Text(welcomeText)
+                .foregroundColor(Color.App.purpleDark)
+
+            Text(onboardingText)
+                .lineLimit(lineLimitForOwnerOnboarding)
+                .foregroundColor(Color.App.purpleDark)
+
+            EditOwnerInformationView(
+                name: $name,
+                surname: $surname,
+                phone: $phone,
+                city: $city,
+                address: $address
+            )
+
+            Button(proceedButtonTitle) {}
+                .padding()
+                .clipShape(Capsule())
+                .disabled(!isSaveButtonEnabled)
```

Comment:
I tried to fill in the mandatory info, but the button was still gray.
Should we hide the prompt about the mandatory fields? 
Does it work properly in your case?
<img width="283" alt="Screenshot 2024-02-23 at 15 26 33" src="https://github.com/IronFoundation/iOS-Woof/assets/32869814/e8b5a205-86d0-47b5-91ad-ea625f5d3a5f">

Reply:
Fixed with e8e2fd4